src/shared/services/jwt/jwt.service.ts

```diff
@@ -0,0 +1,54 @@
+import jwt from 'jsonwebtoken';
+import { DateTime } from 'luxon';
+
+export interface IGenerateTokens_Props {
```

Comment:
Move interfaces to separate files. And change names to `IGenerateTokensInput / Output`